[DO NOT MERGE] Hammonia local

GeneralVonGelre opened the pull request.

emprahtheodosius committed:
KuK Sandzak occupation fix, slight increase of infamy for Russia

GeneralVonGelre committed:
Neutrality Rework

GeneralVonGelre committed:
Slight fix to declaring neutrality

emprahtheodosius committed:
Minor adjustments

GeneralVonGelre committed:
Merge branch 'hammonia-local' into hammonia-local

GeneralVonGelre committed:
Merge pull request #98 from emprahtheodosius/hammonia-local

Le migration/pop tweaks, Crimea, OE, RUS, BUL, Eastern Crisis fixes

GeneralVonGelre committed:
All my homies hate vanilla files

GeneralVonGelre committed:
Immigrants dont mind war as much

GeneralVonGelre committed:
Delete duplicate modifiers

GeneralVonGelre committed:
Healthcare == Expensive

GeneralVonGelre committed:
Education more expensive

GeneralVonGelre committed:
Revert "Education more expensive"

This reverts commit e35526791e6aea6f5c8fe46ce1765b43fc5f256a.

GeneralVonGelre committed:
Revert "Healthcare == Expensive"

This reverts commit f0f9c7f2e7359fdb82316b2865736a396bd35078.

GeneralVonGelre committed:
New World War Update

GeneralVonGelre committed:
Schleswig Holstein Question Fix

GeneralVonGelre committed:
Assim Trouble MTTH Fix

GeneralVonGelre committed:
General plurality nerf

GeneralVonGelre committed:
Theoretical Take Puppet Fix

GeneralVonGelre committed:
Neutrality Fix

GeneralVonGelre committed:
Statehood Decision Spam Fix

GeneralVonGelre committed:
France isnt nearly as attractive

GeneralVonGelre committed:
Health care less effective more expensive

GeneralVonGelre committed:
Disable old Algeria Rebellion

GeneralVonGelre committed:
bureaucrat and tariff nerf

GeneralVonGelre committed:
Schools more expensive

GeneralVonGelre committed:
No take capital CB on neutral countries

GeneralVonGelre committed:
No more puppet Iberia

GeneralVonGelre committed:
No more dual monarchy

GeneralVonGelre committed:
Rebels to vanilla values

GeneralVonGelre committed:
Swiss more tolerant to Latins, less to Luxemburgers

GeneralVonGelre committed:
Replacting state_id with region

GeneralVonGelre committed:
Less lazy Turkey code

GeneralVonGelre committed:
Poznan

GeneralVonGelre committed:
Merge branch 'Developer-Build' into hammonia-local

GeneralVonGelre committed:
Validator Fix

GeneralVonGelre committed:
Posen Uprising (No Localisation)

Hammonia committed:
ottoman pops and thuringia map imrpovement

Hammonia committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
pzn fix

christofergartner committed:
Las Bela can't get annexed

Las Bela could get annexed through the Doctrine of Lapse despite being a Balochi Princely State.

christofergartner committed:
Removed redundant event

The Second Anglo-Burmese War event already exists in East Indies.txt

GeneralVonGelre committed:
Merge pull request #100 from NotNietzsche/patch-1

Las Bela can't get annexed

GeneralVonGelre committed:
New Reform Balance

christofergartner committed:
Industry tech changes

Reverted most to HFM. Some things would just be heretical to revert, so I didn't.
These are: Regenerative furnace in 1836 and Synthetic polymers in 1919. I kept these as-is, 1850 and 1900, respectively.

GeneralVonGelre committed:
Cleaning up Flag folder

GeneralVonGelre committed:
Revert "Cleaning up Flag folder"

This reverts commit c075afd2adea82d842d4a1ada9b90a5504afbf48.

GeneralVonGelre committed:
Revert "Revert "Cleaning up Flag folder""

This reverts commit 19b546491d053a54bf964051b1d18aa72eb01c41.

GeneralVonGelre committed:
Removing dynamic tags

GeneralVonGelre committed:
Dual Monarchy = Gone

Hammonia committed:
better san stefano and ober ost

GeneralVonGelre committed:
Merge pull request #101 from NotNietzsche/patch-2

Removed redundant event

GeneralVonGelre committed:
Merge pull request #102 from NotNietzsche/patch-4

Industry tech changes

GeneralVonGelre committed:
Japan crash fix

christofergartner committed:
Webster-Ashburton fix

Fort William (49) core was kept by the USA even after agreeing to a border.

christofergartner committed:
Bear Flag Revolt fix

The Bear Flag Revolt is no longer forced by the USA doing Manifest Destiny.

christofergartner committed:
Bear Flag Revolt fixed

The Bear Flag Revolt can now only trigger before the USA does Manifest Destiny. This means that California will not become an obstacle to US expansion that 33 Americans are no longer represented as creating a new state.

GeneralVonGelre committed:
Kentucky Fried Code

GeneralVonGelre committed:
Tracking update

GeneralVonGelre committed:
Talleyrand fix + FRA infamy changes

GeneralVonGelre committed:
Mexico Cheaper

GeneralVonGelre committed:
Civil War Fix

christofergartner committed:
CSA Cuba fix

No more purchase during the 150 days of secession and peace.

GeneralVonGelre committed:
Liberals no longer cause guaranteed bankruptcy

GeneralVonGelre committed:
woopsie

christofergartner committed:
Shorter Seminole Wars

10 year MTTH => 5 year MTTH.

christofergartner committed:
Merge remote-tracking branch 'upstream/hammonia-local' into NietzschePatches

christofergartner committed:
Less Seminole spam

16 month MTTH made the event really spammy.

GeneralVonGelre committed:
Relations matter

christofergartner committed:
Merge remote-tracking branch 'upstream/hammonia-local' into NietzschePatches

christofergartner committed:
Maxim gun fix

The Maxim gun invented in the 1870's? Uh-uh, that's a big nono.

christofergartner committed:
Gold Coast

The UK will now always try to purchase all of the Gold Coast to secure it. Whether the Netherlands accepts is a different matter.

christofergartner committed:
Industrial inventions changed

Removed the labourer factor in all inventions. It basically ruined mining. Also added some more modifiers to certain inventions.

GeneralVonGelre committed:
Merge pull request #106 from NotNietzsche/NietzschePatches

USA + industry

GeneralVonGelre committed:
Guyana fix

GeneralVonGelre committed:
French America more accessible

GeneralVonGelre committed:
Statehoodmechanic Localisation

GeneralVonGelre committed:
Woops

emprahtheodosius committed:
Minor FRA fixes, arctic Canada/Alaska, education nerf

GeneralVonGelre committed:
Merge pull request #107 from emprahtheodosius/hammonia-local

Minor FRA fixes, arctic Canada/Alaska, education nerf

GeneralVonGelre committed:
Merge branch 'Developer-Build' into hammonia-local

GeneralVonGelre committed:
Italy as puppets + France infamy fix

GeneralVonGelre committed:
Small update to the issue file

GeneralVonGelre committed:
Localisation fix

GeneralVonGelre committed:
Bracket fix

GeneralVonGelre committed:
Brazil Pop Tempfix

GeneralVonGelre committed:
Second treaty fix

Hammonia committed:
Update positions.txt

GeneralVonGelre committed:
Fix so that my validator works

Elijah-maker committed:
translucent menu background

Hammonia committed:
Smooth Mediterranean Sea

Hammonia committed:
Update positions.txt

Hammonia committed:
Smooth African coast and small fixes

Elijah-maker committed:
RGO things

Elijah-maker committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
Middle east caucasus +

Hammonia committed:
Better Texasgerman names

anzu-3278 commented on TGC/decisions/ACW.txt:
```diff
@@ -1186,4 +1186,53 @@ political_decisions = {
+		}
+
+		effect = {
+			CSA = { country_event = 16401 }
```
Don't forget to fix this before the big merge into dev :D

anzu-3278 commented on TGC/decisions/Italy-Unification.txt:
```diff
@@ -1728,6 +1728,7 @@ political_decisions = {
 					days = 30
 				}
 			}
+			#TODO: Add event for Two Sicilies and the Papal States to propose unification against the Austrian threat
```
I still don't think this would have been particularly likely, though certainly they should get events notifying them and allowing them to turn to France for aid for example.

anzu-3278 commented on TGC/decisions/SPA.txt:
```diff
@@ -1848,119 +1848,7 @@ political_decisions = {
 	}
-
-#from more decisions.txt
-	form_iberia = {
```
Now that we have an option to disable these this removal should be reverted.

anzu-3278 committed:
Merge branch 'Developer-Build' into hammonia-local

Hammonia committed:
map fix

Hammonia committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

GeneralVonGelre committed:
Statehood Hotfix

anzu-3278 committed:
Renaming decisions for Finland, Karelia, Ingria and the Baltics + state ids and capitals fixed

anzu-3278 committed:
Sivas moved to the correct state

GeneralVonGelre committed:
Return of the missing flags

GeneralVonGelre committed:
Adding Monarchy 2 flagtype

GeneralVonGelre committed:
Setup for removal of Bourbon and Carlist tag

GeneralVonGelre committed:
Carlist Spain & Bourbon France removed

GeneralVonGelre committed:
Merge branch 'Developer-Build' into hammonia-local

GeneralVonGelre committed:
Sao Paulo missing pop fix

GeneralVonGelre committed:
Whole bunch of validator fixes

anzu-3278 committed:
Poland renaming decisions

GeneralVonGelre committed:
Some african core fixes

GeneralVonGelre committed:
French Alliance fix

GeneralVonGelre committed:
Italy + Dutch Indonesia fix

GeneralVonGelre committed:
Add Substate Colony CB

GeneralVonGelre committed:
More restrictive CBs

GeneralVonGelre committed:
Removing empty files

GeneralVonGelre committed:
Talleyrand Fix

Hammonia committed:
South Asia & Oceania Coastline fix

Hammonia committed:
Indian Exclaves

anzu-3278 committed:
wip

anzu-3278 committed:
nits

Hammonia committed:
Rest of Eurasia and Alaska

anzu-3278 committed:
Started moving Greek renaming to new decisions

Hammonia committed:
Kaspian and Rivers

Hammonia committed:
gibraltar fix

anzu-3278 committed:
okay it doesn't crash please don't revert anything now

anzu-3278 committed:
Lithuania and Vilnius are now the right way around

Hammonia committed:
Update GEF.txt

GeneralVonGelre committed:
Release Nation bandaid fix

anzu-3278 committed:
Soviet Names, Romanian Podolia names, Ararat terrain, preparation for Eastern Anatolia names

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
CAN+USA Border and Coastline Clean-Up

anzu-3278 committed:
Ukraine and Soviet renaming

anzu-3278 committed:
Sea of Azov rename and fixes

anzu-3278 committed:
Anatolia renaming

Hammonia committed:
finish lakes south america coastal fix and new Belize/Yucatan provs

Hammonia committed:
names mexblz

Hammonia committed:
LATAMPOPS

Hammonia committed:
Update colormap_political.dds

Hammonia committed:
no owner in blz

Hammonia committed:
wee

anzu-3278 committed:
Armenian Azerbaijan

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
tropics = tropical wood and Kniphausen + smolfix

Hammonia committed:
map fixes

anzu-3278 committed:
Romania renaming decisions

anzu-3278 committed:
Renaming decisions for Transylvania, Hungary, Galicia

Hammonia committed:
Indian map changes and central american pops and names

anzu-3278 committed:
Clayton-Bulwer Treaty sans localization

anzu-3278 committed:
minor validator fixes

anzu-3278 committed:
HFM style culture colors submod (not yet working)

Hammonia committed:
GFX-Fix

Hammonia committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

anzu-3278 committed:
Czechoslovakia and Burgenland renaming decisions

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

anzu-3278 committed:
Slovenia, Croatia, Vojvodina

anzu-3278 committed:
Serbia, Montenegro, Bosnia

anzu-3278 committed:
Venice renaming + a few fixes

anzu-3278 committed:
Italy renaming decisions

Hammonia committed:
Indian pops + Central america

anzu-3278 committed:
wip

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

anzu-3278 committed:
pops issues fixed

Hammonia committed:
Update India.txt

Hammonia committed:
Revert "Update India.txt"

This reverts commit 823199cb72f735b49b5cff05a89eb023e077b73a.

Hammonia committed:
Revert "Revert "Update India.txt""

This reverts commit 59eb174b1b02ca32e6d7ffeca0fdfb943b495883.

Hammonia committed:
Update India.txt

Hammonia committed:
rest of india fix

anzu-3278 committed:
readding reverted changes

anzu-3278 committed:
Bulgaria, Macedonia, Thrace renaming decisions

Hammonia committed:
cosguapops

anzu-3278 committed:
Southwestern Macedonia names according to province shape

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
FIX

Hammonia committed:
Update 00_map-provinces.csv

Hammonia committed:
Update 00_map-provinces.csv

anzu-3278 committed:
German Unification fixes, now account for German Confederation membership mechanic, plus few nitpicks

anzu-3278 committed:
miscellaneous fixes

anzu-3278 committed:
random renaming todos fixed

anzu-3278 committed:
Greece, Albania, Austria renaming decisions

anzu-3278 committed:
Moving the Statehood Mechanic to a submod

anzu-3278 committed:
Belize purchase readjusted + some nitpicks

Hammonia committed:
minor adjustments

ColdSlav committed:
Chile Political Parties Fixes

Fixed unlocalized party names, removed old parties, & added some missing parties

anzu-3278 committed:
Merge pull request #121 from ColdSlav/hammonia-local

Chile's Political Parties Fixes

anzu-3278 committed:
Anatolia renaming decisions

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

ColdSlav committed:
Argentina's Political Parties Overhaul + Some Fixes

Added more political parties to Argentina and fixed some issues present on Chile's political parties.

anzu-3278 committed:
Middle east renaming decisions

anzu-3278 committed:
Germany renaming decisions and fixes

anzu-3278 committed:
remove Kniphausen cores after annexing

anzu-3278 committed:
MTC adjectives fixed

Hammonia committed:
dumb railroad fixes

Hammonia committed:
Update positions.txt

Hammonia committed:
Update positions.txt

anzu-3278 committed:
few fixes

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

anzu-3278 committed:
Preparation for Caste War content + updates to South America pops and starting leaders

Hammonia committed:
adjustments

ColdSlav committed:
Merge remote-tracking branch 'upstream/hammonia-local' into hammonia-local

ColdSlav committed:
Peru's Political Parties Overhaul

More parties what else lol

ColdSlav committed:
Bolivia's Political Parties Overhaul

you know

ColdSlav committed:
Paraguay's Parties Done

what else do you expect me to put here lol

anzu-3278 committed:
Schleswig War German Unification fix

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

Hammonia committed:
Update 00_map-provinces.csv

champfish committed:
Implemented Filibuster Wars

champfish committed:
Merge pull request #1 from rderekp/hammonia-local

Update from origin

champfish committed:
minor text fixes

champfish committed:
Merge branch 'hammonia-local' of https://github.com/champfish/The-Grand-Combo into hammonia-local

champfish committed:
minor text edit 2

champfish committed:
Split FIL content into own file

anzu-3278 committed:
Preparation and plans for new Central America content

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

anzu-3278 committed:
Denmark event province fixes

anzu-3278 committed:
Renaming decisions localization + random fixes

Fraggy1407 committed:
almost done

Fraggy1407 committed:
Added indentations for better readability

Fraggy1407 committed:
tied up some loose ends

Fraggy1407 committed:
Merge branch 'hammonia-local' into Great-Hungarian-Game

champfish committed:
Nicaragua Filibuster Fixes

ColdSlav committed:
Ecuador Political Parties + Smol Argentina Fix

DFOIVJGVUIEDDSJI0V9SJ (what else do i put here lmao)

ColdSlav committed:
Colombia and Venezuela's Parties Done

BIBOBOBBIBIBIOBOBOBKASNIASDIUUUUUUUUUUUUUUUUUUUUUUUUASSDOVUJN
:)

ColdSlav committed:
Brazil's Political Parties

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

ColdSlav committed:
Brazil Fixes

OOOOOOOOOOOOO

ColdSlav committed:
Uruguay's Parties Done

I DID IT

?

anzu-3278 committed:
Persia localizations and errors

ColdSlav committed:
Chile's Parties Improvements

Added the socialist faction of the Partido Radical, the Partido Radical economic policy shift in the 10's and some smol adjustments.

anzu-3278 committed:
Clayton-Bulwer Treaty localizations

anzu-3278 committed:
Merging Dev into Hammonia

anzu-3278 committed:
Random Validator fixes and removing references to the deprecated BOR tag

ColdSlav committed:
.gitignore fix

should be good now (I suppose)

champfish committed:
FIL gets US tech + minor edits

champfish committed:
One new line

Fraggy1407 committed:
Formating fixed

anzu-3278 committed:
Merge pull request #126 from ColdSlav/hammonia-local

South America's Political Parties Overhaul

anzu-3278 committed:
misc fixes

anzu-3278 committed:
Merge branch 'hammonia-local' of https://github.com/rderekp/The-Grand-Combo into hammonia-local

champfish committed:
Validator fixes

anzu-3278 committed:
Merge pull request #125 from champfish/hammonia-local

Add Nicaragua Filibuster Content

anzu-3278 committed:
William Walker traits fix

anzu-3278 committed:
Lithuanian rename for Brest

anzu-3278 committed:
csa capital fix

champfish committed:
Fixed FIL's event to join CSA

GeneralVonGelre committed:
Merge pull request #131 from champfish/patch-1

Fixed FIL's event to join CSA

anzu-3278 committed:
Bully Gelre until he starts writing localizations for his events

anzu-3278 committed:
Merge branch 'hammonia-local' into Great-Hungarian-Game

anzu-3278 committed:
Merge pull request #124 from rderekp/Great-Hungarian-Game

Great hungarian game

anzu-3278 committed:
minor syntax issues in GHG

anzu-3278 committed:
Merge branch 'Developer-Build' into hammonia-local
